internal/pkg/template/workload.go

```diff
@@ -392,67 +385,45 @@ type WorkloadOpts struct {
 	ScheduleExpression string
 	StateMachine       *StateMachineOpts
 
-	// Additional options for worker service templates.
-	Subscribe *SubscribeOpts
-}
-
-// ParseRequestDrivenWebServiceInput holds data that can be provided to enable features for a request-driven web service stack.
-type ParseRequestDrivenWebServiceInput struct {
-	Variables           map[string]string
-	StartCommand        *string
-	Tags                map[string]string        // Used by App Runner workloads to tag App Runner service resources
-	NestedStack         *WorkloadNestedStackOpts // Outputs from nested stacks such as the addons stack.
-	AddonsExtraParams   string                   // Additional user defined Parameters for the addons stack.
-	EnableHealthCheck   bool
-	EnvControllerLambda string
-	Publish             *PublishOpts
-	Platform            RuntimePlatformOpts
-
+	// Additional options for request driven web service templates.
+	StartCommand      *string
+	EnableHealthCheck bool
 	// Input needed for the custom resource that adds a custom domain to the service.
 	Alias                *string
 	ScriptBucketName     *string
 	CustomDomainLambda   *string
 	AWSSDKLayer          *string
 	AppDNSDelegationRole *string
 	AppDNSName           *string
+
+	// Additional options for worker service templates.
+	Subscribe *SubscribeOpts
 }
 
 // ParseLoadBalancedWebService parses a load balanced web service's CloudFormation template
 // with the specified data object and returns its content.
 func (t *Template) ParseLoadBalancedWebService(data WorkloadOpts) (*Content, error) {
-	if data.Network == nil {
```

Comment:
Quick question b/c I'm likely missing something: what do we do with the defaults now?

Reply:
This is previously a duplicated behavior as the default has already been handled in the converter.